src/sentry/workflow_engine/processors/workflow.py

```diff
@@ -72,6 +73,43 @@ def delete_workflow(workflow: Workflow) -> bool:
     return True
 
 
+@dataclass
+class DelayedWorkflowV2Item:
+    workflow: Workflow
+    delayed_when_conditions: list[DataCondition]
+    delayed_if_conditions: list[DataCondition]
+    event: GroupEvent
+    passing_if_groups: list[DataConditionGroup]
+    timestamp: datetime
+
+    def buffer_key(self) -> str:
+        when_condition_group_set = {
+            condition.condition_group_id for condition in self.delayed_when_conditions
+        }
+        if_condition_group_set = {
+            condition.condition_group_id for condition in self.delayed_if_conditions
+        }
+        when_condition_groups = ",".join(
+            str(condition_group_id) for condition_group_id in sorted(when_condition_group_set)
+        )
+        if_condition_groups = ",".join(
+            str(condition_group_id) for condition_group_id in sorted(if_condition_group_set)
+        )
+        passing_if_group_ids = ",".join(
+            str(condition_group.id) for condition_group in self.passing_if_groups
+        )
+        return f"{self.workflow.id}:{self.event.group.id}:{when_condition_groups}:{if_condition_groups}:{passing_if_group_ids}"
```

Comment:
now we will enqueue the condition groups that have slow conditions + IF condition groups that should fire if the when condition group passes